platform/broker/src/main.rs

```diff
@@ -0,0 +1,28 @@
+use apca::api::v2::orders;
+use apca::{ApiInfo, Client};
+
+#[tokio::main]
+async fn main() {
+    let api_info = match ApiInfo::from_env() {
+        Ok(api_info) => api_info,
+        Err(e) => {
+            panic!("Failed to get API info: {:?}", e);
+        }
+    };
+    let client = Client::new(api_info);
+
+    let request = orders::ListReq {
+        status: orders::Status::Closed,
+        limit: Some(100),
+        _non_exhaustive: (),
+        symbols: vec!["JNJ".to_string()],
```

Comment:
This looks temporarily hard-coded. Where will the values be pulled from?

Reply:
probably we'll put this in a knative service and have a `GET /list/orders/{ticker}`

Reply:
for now just have this as a working skeleton

Reply:
https://github.com/pocketsizefund/pocketsizefund/issues/315